### PR TITLE
add 'procedure_duration' column, fill up repository interfaces, add repository impl

### PR DESCRIPTION
**Pull request by eugene-mytrofanov**
I forgot to add 'procedure_duration' column to the 'medical_procedures' table. It was fixed in this branch.
Could you check repository interfaces and classes?

### CODE REVIEW

File: src/main/java/ua/su/repository/impl/ClinicRepositoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public Clinic getOne(Long id) {
+        return jdbcTemplate.queryForObject("SELECT * FROM clinics WHERE id = ?", ROW_MAPPER, id);
+    }
+
+    @Override
+    public Clinic insert(Clinic clinic) {
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps =
+                    connection.prepareStatement("insert into clinics(name, address, phone, is_insurance_supported, " +
```
Comment (Andriireva) [inline src/main/java/ua/su/repository/impl/ClinicRepositoryImpl.java]: Here sql is in a lower case, but in the getOne() method is upper case. It is better to use same sql everywhere.

File: src/main/java/ua/su/repository/impl/MedicalProcedureRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    public MedicalProcedure getOne(Long id) {
+        return jdbcTemplate.queryForObject("SELECT * FROM medical_procedures WHERE id = ?", ROW_MAPPER, id);
+    }
+
+    @Override
+    public MedicalProcedure insert(MedicalProcedure medicalProcedure, Integer clinicId) {
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps =
+                    connection.prepareStatement("insert into medical_procedures(title, price, insurance_coverage, " +
```
Comment (Andriireva) [inline src/main/java/ua/su/repository/impl/MedicalProcedureRepositoryImpl.java]: sql are in different cases

**Review by Andriireva — CHANGES_REQUESTED** [review]

File: src/main/java/ua/su/repository/impl/ClinicRepositoryImpl.java (hunk 1 above)
Comment (eugene-mytrofanov) [reply src/main/java/ua/su/repository/impl/ClinicRepositoryImpl.java]: I modified SQL statements to the upper case.

File: src/main/java/ua/su/repository/impl/MedicalProcedureRepositoryImpl.java (hunk 2 above)
Comment (eugene-mytrofanov) [reply src/main/java/ua/su/repository/impl/MedicalProcedureRepositoryImpl.java]: modified as well

**Review by Andriireva — APPROVED** [review]

**Pull request merged**